src/examples/PointTarget/Output/Ns3/QdFiles/qdOutput.json

```diff
@@ -1,2 +1,2 @@
[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.64159326E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.6413284E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.64106443E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.64080118E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.64053863E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.64027715E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.64001621E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63975615E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63949698E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63923852E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63898094E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63872408E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63846811E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63821285E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63795847E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63770481E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63745203E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.6372E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63694879E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63669833E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63644875E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.6362E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63595208E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63570481E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63545843E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63521276E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63496798E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.30386146E-8,2.30673329E-8,2.35733335E-8,2.63472408E-8,2.96216882E-8,3.23998819E-8,3.24854561E-8],[2.3E-8,2.303
```

Comment:
You can replace them since I rerun some of the scenarios

Reply:
Did you run the scenario changing some of the settings? If that's the case, I agree with including it, but I don't see any change in the Input folder. 
What I meant is that every time the scenario is rerun, some minor changes due to the finite numeric precision will occur, that are not relevant and do not add/change anything meaningful to the example. I would only include substantial changes to the scenario, e.g., if new settings or new targets were used, or if the environment was modified, or if an error is found in the output files. Otherwise, every time we rerun them and negligible numerical changes occur in the output, we have a new commit and eventually a very long history of non-relevant changes.

Of course, if I misunderstood this commit and it is actually a correction or a significant change, we can include it. Please let me know your opinion on this and if I've been clear

Reply:
Okay, please do whatever is necessary.